base/array.jl

```diff
@@ -752,9 +837,45 @@ function findnext(A, v, start::Integer)
     end
     return 0
 end
+"""
+    findfirst(A, v)
+
+Return the linear index of the first element equal to `v` in `A`.
+Returns 0 if `v` is not found.
```

Comment:
It wasn't *me* who put the 0s in backticks :). I updated some of these docstrings which didn't contain this info and didn't include the ticks. Personally I like the appearance of the "raw" 0 better, but from a consistency with `i` point of view, I could see escaping the 0s. Anyone else have thoughts?